app/javascript/src/dashboard.js

```diff
@@ -268,6 +268,107 @@ $('document').ready(() => {
     return true
   })
 
+  // Add Supervisors Table
+  const supervisorsTable = $('table#supervisors').DataTable({
+    autoWidth: false,
+    stateSave: false,
+    order: [[1, 'asc']], // order by cast contacts
+    columns: [
+      {
+        name: 'display_name',
+        className: 'min-width',
+        render: (data, type, row, meta) => {
+          return `
+            <a href="${editSupervisorPath(row.id)}">
+              ${row.display_name || row.email}
+            </a>
+          `
+        }
+      },
+      {
+        name: '',
+        className: 'min-width',
+        render: (data, type, row, meta) => {
+          const noContactVolunteers = Number(row.no_attempt_for_two_weeks)
+          const transitionAgedCaseVolunteers = Number(row.transitions_volunteers)
+          const activeContactVolunteers = Number(row.volunteer_assignments) - noContactVolunteers
+          const activeContactElement = activeContactVolunteers
+            ? (
+            `
+            <span class="attempted-contact status-btn success-bg text-white pl-${activeContactVolunteers * 15} pr-${activeContactVolunteers * 15}">
+              ${activeContactVolunteers}
+            </span>
+            `
+              )
+            : ''
+
+          const noContactElement = noContactVolunteers > 0
+            ? (
+            `
+            <span class="no-attempted-contact status-btn danger-bg text-white pl-${noContactVolunteers * 15} pr-${noContactVolunteers * 15}">
+              ${noContactVolunteers}
+            </span>
```

Comment:
Is there a way to keep all of this HTML in the view, rather than in JS? Can we toggle classes to hide/unhide instead?

Reply:
I would love to switch this over to Stimulus! Do you have experience with that?

Reply:
> I would love to switch this over to Stimulus! Do you have experience with that?

I have some experience with it, and it's something we can look into for sure. I think if we were to build something, we could use it for both the supervisor and volunteer index pages where we want the ability to filter dynamically on the frontend.